api/src/main/java/jakarta/data/repository/Select.java

```diff
@@ -25,23 +25,27 @@
 import java.lang.annotation.Target;
 
 /**
- * <p>Determines which entity attribute(s) are returned by a {@link Find}
- * operation.</p>
+ * <p>Determines the attribute or attributes of an entity which are returned
+ * by a {@linkplain Find parameter-based} or {@linkplain Query annotated}
+ * query method.</p>
  *
- * <p>This annotation can be specified on a repository find method or
- * on a record component.</p>
+ * <p>This annotation may be applied to a repository method annotated
+ * {@code @Find}, or to a component of a record type.</p>
```

Comment:
Minor grammar correction:
```suggestion
 * {@code @Find} or to a component of a record type.</p>
```